Assets/Scripts/NPCs/DummyPlayer.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DummyPlayer : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
```

Comment:
```suggestion
```

Reply:
DummyPlayer is simply a class I used for checking some things. It won't be used in the final product.